jbpm-flow/src/main/java/org/jbpm/workflow/instance/impl/NodeInstanceImpl.java

```diff
@@ -393,7 +393,7 @@ public void triggerNode(long nodeId) {
     	org.jbpm.workflow.instance.NodeInstance nodeInstance = (org.jbpm.workflow.instance.NodeInstance)
     		((org.jbpm.workflow.instance.NodeInstanceContainer) getNodeInstanceContainer())
             	.getNodeInstance(getNode().getNodeContainer().getNode(nodeId));
-    	triggerNodeInstance(nodeInstance, null);
+    	triggerNodeInstance(nodeInstance, org.jbpm.workflow.core.Node.CONNECTION_DEFAULT_TYPE);
```

Comment:
@mswiderski what is the reason for changing this?  I would expect the argument to indicate why the node was triggered, in this case the node is triggered explicitly by the user, not through an incoming connection, so wouldn't null be more adequate (to distinguish from incoming connection)?